api/src/play_asset_pack_manager.gd

```diff
@@ -99,3 +99,35 @@ func get_pack_location(pack_name : String) -> PlayAssetPackLocation:
 	if query_dict == null:
 		return null
 	return PlayAssetPackLocation.new(query_dict)
+
+# -----------------------------------------------------------------------------
+# Returns the location of all installed asset packs. More specifically, a 
+# Dictionary, where for each entry, the key is the asset pack anme and value is 
+# the corresponding PlayAssetLocation object.
+# -----------------------------------------------------------------------------
+func get_pack_locations() -> Dictionary:
```

Comment:
Unfortunately static typing of Dictionary and Array is not support for GDScript, so we can not enforce the exact type of object within this Dictionary.